jupyterhub/spawner.py

```diff
@@ -1618,5 +1618,7 @@ def user_env(self, env):
         return env
 
     def move_certs(self, paths):
-        """No-op for installing certs"""
+        """
+        No-op for installing certs
+        """
         return paths
```

Comment:
I think it also wants for the file to end with an empty line.
```suggestion
        return paths

```

Reply:
No worries @GeorgianaElena :) Thanks for thinking about this with me!